### PR TITLE
Feature/add object state change time

### PR DESCRIPTION
**Pull request by raviks789**
ref #53

### CODE REVIEW

File: library/Jira/MonitoringInfo.php [hunk 1]
```diff
@@ -121,6 +122,19 @@ public function getStateName()
...
     }
 
+    public function getLastStateChange()
+    {
+        if ($this->object instanceof MonitoredObject) {
+            if ($this->object instanceof Service) {
+                return DateFormatter::formatDateTime($this->object->last_state_change);
+            }
+
+            return DateFormatter::formatDateTime($this->object->last_state_change);
```
Comment (nilmerg) [inline library/Jira/MonitoringInfo.php:132]: You access the same property in the same way in both cases. Why the distinction?

File: library/Jira/MonitoringInfo.php [hunk 2]
```diff
@@ -213,6 +223,8 @@ public function getDescriptionHeader()
 
         $description .= sprintf("Host: %s\n", $hostLink);
 
+        $description .= sprintf("Last icinga state change: %s\n", $this->getLastStateChange());
```
Comment (sukhwinder33445) [inline library/Jira/MonitoringInfo.php:226]: Please remove "icinga". "Last state change:" is sufficient.

Comment (lippserd) [reply library/Jira/MonitoringInfo.php:226]: Wouldn't this also have to be translated?

Comment (lippserd) [reply library/Jira/MonitoringInfo.php:226]: Of course only if this is somehow displayed in Web. If this information is only transferred to Jira, this makes no sense :).

**Review by sukhwinder33445 — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**